etc/config-template.toml

```diff
@@ -64,6 +64,13 @@
 ## If not set, `addr` will be used.
 # advertise-addr = ""
 
+## If HTTP service is enabled. If the value is true, the Prometheus can pull the metrics directly 
+## through the HTTP address. Otherwise, the Prometheus will get the metrics through the PushGateway.
+# http-enabled = false
```

Comment:
remove this flag. We can check whether HTTP is enabled or not by http addr config.

Reply:
Emmm, without this flag, how do we know if we need to use the default address?

Reply:
I think it can be removed. Use an empty http address to disable the configuration.